azdev/utilities/const.py

```diff
@@ -6,10 +6,35 @@
 
 import sys
 
+ACTIVATE_PS = 'Activate.ps1'
+AZ_AZDEV_DIR = 'AZDEV_CONFIG_DIR'
+AZ_CLOUD = 'AzureCloud'
+AZ_CONFIG_DIR = 'AZURE_CONFIG_DIR'
+AZ_DEV_SRC = 'dev_sources'
+BASH_EXE = '/bin/bash'
+BASH_NAME_WIN = 'bash.exe'
+BAT_ACTIVATE = 'activate.bat'
+CLI_SECTION = 'clipath'
+CLOUD_SECTION = 'cloud'
```

Comment:
Those global constant string are used not very frequently, most of them are used no more than 2 times. No need to create

```suggestion
```

Besides, adding a prefix to a group of global constant strings would be better to read and indicate its usage at first glimpse.

Reply:
To make const values clean, I will remove all consts which used no more than 2 times.